src/main/java/org/takes/facets/auth/codecs/CcAES.java

```diff
@@ -145,6 +120,36 @@ public Identity decode(final byte[] bytes) throws IOException {
     }
 
     /**
+     * Create AlgorithmParameterSpec with the block size.
+     *
+     * @return The AlgorithmParameterSpec
+     */
+    private static AlgorithmParameterSpec initAlgorithmParameterSpec() {
```

Comment:
@sebing check this document: http://www.yegor256.com/images/books/elegant-objects/seven-pages.pdf